src/qibolab/instruments/qblox/cluster_qcm_bb.py

```diff
@@ -170,99 +157,61 @@ def connect(self):
-                self._set_device_parameter(target, "offset_awg_path0", value=0)
-                self._set_device_parameter(target, "offset_awg_path1", value=0)
-                self._set_device_parameter(target, "sync_en", value=False)  # Default after reboot = False
-                self._set_device_parameter(target, "upsample_rate_awg_path0", "upsample_rate_awg_path1", value=0)
-
-            for target in [
-                self.device.sequencers[self.DEFAULT_SEQUENCERS["o1"]],
-                self.device.sequencers[self.DEFAULT_SEQUENCERS["o2"]],
-                self.device.sequencers[self.DEFAULT_SEQUENCERS["o3"]],
-                self.device.sequencers[self.DEFAULT_SEQUENCERS["o4"]],
-            ]:
-                self._set_device_parameter(target, "connect_out0", value="off")  # Default after reboot = True
-                self._set_device_parameter(target, "connect_out1", value="off")  # Default after reboot = True
-                self._set_device_parameter(target, "connect_out2", value="off")  # Default after reboot = True
-                self._set_device_parameter(target, "connect_out3", value="off")  # Default after reboot = True
-
-            self._set_device_parameter(
-                self.device.sequencers[self.DEFAULT_SEQUENCERS["o1"]],
-                "connect_out0",
-                value="I",
-            )
-            self._set_device_parameter(
-                self.device.sequencers[self.DEFAULT_SEQUENCERS["o2"]],
-                "connect_out1",
-                value="Q",
-            )
-            self._set_device_parameter(
-                self.device.sequencers[self.DEFAULT_SEQUENCERS["o3"]],
-                "connect_out2",
-                value="I",
-            )
-            self._set_device_parameter(
-                self.device.sequencers[self.DEFAULT_SEQUENCERS["o4"]],
-                "connect_out3",
-                value="Q",
-            )
+                target.set("cont_mode_en_awg_path0", False)  # Default after reboot = False
+                target.set("cont_mode_en_awg_path1", False)
+                target.set("cont_mode_waveform_idx_awg_path0", 0)
+                target.set("cont_mode_waveform_idx_awg_path1", 0)
+                target.set("marker_ovr_en", True)
+                target.set("marker_ovr_value", 15)
+                target.set("mixer_corr_gain_ratio", 1)
+                target.set("mixer_corr_phase_offset_degree", 0)
+                target.set("offset_awg_path0", 0)
+                target.set("offset_awg_path1", 0)
+                target.set("sync_en", False)
+                target.set("upsample_rate_awg_path0", 0)
+                target.set("upsample_rate_awg_path1", 0)
+                target.set("connect_out0", "off")  # Default after reboot = True
+                target.set("connect_out1", "off")
+                target.set("connect_out2", "off")
+                target.set("connect_out3", "off")
```

Comment:
It's rather simpler if you just define a dictionary with the map, and then you loop over it
```py
for name, value in defaults.items():
    target.set(name, value)
```

Reply:
Yes, I wanted to do that however I can see better this dictionary in the other PR (where we wrote the file `module.py`) since there will be many keys in common. Btw I will implement it here as top-level constant and then we can bring this to the other branch.

Reply:
I opened an issue about this, for now I mark it as resolved.